WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+MoreActions.swift

```diff
@@ -12,13 +12,12 @@ extension GutenbergViewController {
     func displayMoreSheet() {
         let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
 
-        //TODO: Comment in when bridge is ready
-        /*if mode == .richText {
+        if mode == .richText, let contentInfo = contentInfo {
             // NB : This is a candidate for plurality via .stringsdict, but is limited by https://github.com/wordpress-mobile/WordPress-iOS/issues/6327
-            let textCounterTitle = String(format: NSLocalizedString("%li words, %li characters", comment: "Displays the number of words and characters in text"), richTextView.wordCount, richTextView.characterCount)
+            let textCounterTitle = String(format: NSLocalizedString("%li block(s), %li word(s), %li character(s)", comment: "Displays the number of blocks, words and characters in text"), contentInfo.wordCount, contentInfo.characterCount)
```

Comment:
The variable for `%li block(s)` seems to be missing here. It gives wrong numbers to the display.

Reply:
Good catch!

Reply:
I fixed it, do you mind give it another go @etoledom ?